tests/ert/ui_tests/gui/test_manage_experiments_tool.py

```diff
@@ -547,3 +548,74 @@ def test_realization_view(
         "BPR_555_PERSISTENCE - PARAMETERS_LOADED",
         "BPR_138_PERSISTENCE - PARAMETERS_LOADED",
     } == set(realization_widget._parameter_text_edit.toPlainText().strip().splitlines())
+
+
+def test_parameters_pane(
```

Comment:
maybe more concrete suggestion in test name:
`test_parameters_pane_has_the_correct_entries` or something along those lines.